cmd/buildah/from.go

```diff
@@ -12,6 +12,10 @@ import (
 var (
 	fromFlags = []cli.Flag{
 		cli.StringFlag{
+			Name:  "authfile",
+			Usage: "Path of the authentication file. Default is ${XDG_RUNTIME_DIR}/containers/auth.json",
```

Comment:
Path to path.  If you could also lowercase tls verbiage while your in here, you'd fix up my oopsie.

Reply:
fixed